src/test/regress/expected/multi_explain.out

```diff
@@ -452,10 +452,10 @@ Aggregate
                                 Sort Key: ((users.composite_id).tenant_id), ((users.composite_id).user_id)
                                 ->  Hash Join
                                       Hash Cond: (users.composite_id = events.composite_id)
-                                      ->  Seq Scan on users_1400029 users
+                                      ->  Seq Scan on users_1400033 users
```

Comment:
What triggered this change?

Reply:
This happened after I create another table called agg_results_window for the window function tests. I added this table because this test was using the same table with another test which they run in parallel right now.